chart/kubeapps/templates/kubeapps-frontend-deployment.yaml

```diff
@@ -39,6 +39,7 @@ spec:
     {{- if .Values.authProxy.enabled }}
       - name: auth-proxy
         args:
+        - keycloak-proxy
```

Comment:
You don't need this change using `bitnami/keycloak-gatekeeper:2.3.0-scratch-r1` on

Reply:
actually it's needed. And now the path of the binary `/keycloack-proxy` is not in the path so I needed to specify the full path. Note that for a pod to inherit a "command" you need to set it as `ENTRYPOINT` of the image (not `CMD` what may seems the logic approach).